### ISSUE TITLE
Simulation output to memory?

### ISSUE THREAD

**Issue by tinrabuzin** [OpenModelica/OpenModelica#7319]
Is there any way or plans to avoid writing simulation results to a file and instead having it in the memory? 

Specifically, I'm using OMPython and noticed that my code spends a lot of time accessing the simulation results file (compared to the simulation time). And that happens even when I use regular expressions to select only two variables to be written to the output. 

I realize that having the simulation results in the memory is problematic when the results are of large size but for smaller models it can be feasible...

**Comment by lochel** [comment]
We have such plans for [OMSimulator](https://github.com/OpenModelica/OMSimulator) and OMEdit to connect directly to the simulation via zmq to read and display the simulation results. Maybe you are interested in something similar? You are welcome to provide more input and use-cases.

**Comment by adeas31** [comment]
Is already possible via interactive simulation. I guess you need to implement a OPC-UA client in python for that and you need to run the simulation in OMPython with [embeddedserver flag](https://openmodelica.org/doc/OpenModelicaUsersGuide/latest/simulationflags.html?highlight=embeddedserver#simflag-embeddedserver).

@arun3688 it is possible to specify simulation flags in OMPython, right?

**Comment by arun3688** [comment]
@adeas31  yes, if you are using `ModelicaSystem`  constructor then you can use simulation flag something like below

```
from OMPython import ModelicaSystem
mod=ModelicaSystem(model_path + "BouncingBall.mo","BouncingBall")
mod.simulate(simflags="-noEventEmit -noRestart -override=e=0.3,g=9.71") // method-3 simulationflags provided by users
```
For more information see https://www.openmodelica.org/doc/OpenModelicaUsersGuide/latest/ompython.html#enhancedompython

**Comment by lochel** [comment]
There is also an easier tcp-based output for the simulation executable. It send the simulation results as text string via a siple socket connection.

**Comment by tinrabuzin** [comment]
Thank you all for the suggestions. I will try them out.
